Source/RxCBCharacteristic.swift

```diff
@@ -33,10 +33,13 @@ class RxCBCharacteristic: RxCharacteristicType {
     }
 
     var uuid: CBUUID {
-        return characteristic.UUID
+        return characteristic.uuid
     }
-    var value: NSData? {
-        return characteristic.value
+    var value: Data? {
+        if let value = characteristic.value {
+            return value
+        }
+        return nil
```

Comment:
Yes, also changed `descriptors` to use optional chaining so the guard statement can be omitted.